src/modules/LanUdevAuto.rb

```diff
@@ -32,11 +32,15 @@
 
 module Yast
   class LanUdevAutoClass < Module
+
+    include Logger
```

Comment:
I think it is better to be more explicit and include Yast::Logger as Logger is well known ruby class so to avoid confusion